### ISSUE TITLE
Add a LICENSE for the project

### ISSUE THREAD

**Issue by ahachete** [arlind29/KoalaBench#3]
An open source license would be welcomed :)

**Comment by arlind29** [comment]
The original licence of TPC-H is Apache License, Version 2.0. The same licence should be used on any edited version.

**Comment by ahachete** [comment]
Agreed. That is a nice license for the project. I'd then add it to a LICENSE file in the root of the directory and the header of every source code file in the project :)
